apps/wallet-mobile/src/WalletNavigator.tsx

```diff
@@ -151,28 +153,38 @@ const WalletTabNavigator = () => {
 }
 
 const Stack = createStackNavigator<WalletStackRoutes>()
-export const WalletNavigator = () => (
-  <Stack.Navigator
-    screenOptions={{
-      headerShown: false /* used only for transition */,
-      detachPreviousScreen: false /* https://github.com/react-navigation/react-navigation/issues/9883 */,
-    }}
-  >
-    <Stack.Screen name="wallet-selection" component={WalletSelectionScreen} />
+export const WalletNavigator = () => {
+  const {initialUrl} = useInitialLink()
+  const {resetToWalletSelection} = useWalletNavigation()
 
-    <Stack.Screen name="main-wallet-routes" component={WalletTabNavigator} />
+  return (
+    <Stack.Navigator
+      initialRouteName={initialUrl !== null ? 'exchange-result' : 'wallet-selection'}
```

Comment:
here is the route after login

Reply:
@michaeljscript i think there will be a conflict with your changes. i would say that the result screen is the priority

Reply:
product question i think

Reply:
wait you are using a different navigator i think